### DISCUSSION TITLE
v2.1.6 be实际使用内存大于be认为的内存

### DISCUSSION THREAD

**Discussion by peiyaqi** [apache/doris#46145] [A - General / Q&A]
I20241227 13:43:19.271855 62667 daemon.cpp:221] os physical memory 62.60 GB. process memory used 54.06 GB(= 54.83 GB[vm/rss] - 782.24 MB[tc/jemalloc_cache] + 0[reserved] + 0B[waiting_refresh]), limit 56.34 GB, soft limit 50.71 GB. sys available memory 7.67 GB(= 7.67 GB[proc/available] - 0[reserved] - 0B[waiting_refresh]), low water mark 3.13 GB, warning water mark 6.26 GB.
I20241227 13:43:19.582617 62667 daemon.cpp:221] os physical memory 62.60 GB. process memory used 54.29 GB(= 55.13 GB[vm/rss] - 859.28 MB[tc/jemalloc_cache] + 0[reserved] + 0B[waiting_refresh]), limit 56.34 GB, soft limit 50.71 GB. sys available memory 6.79 GB(= 6.79 GB[proc/available] - 0[reserved] - 0B[waiting_refresh]), low water mark 3.13 GB, warning water mark 6.26 GB.
    os physical memory 62.60 GB. process memory used 54.71 GB(= 55.38 GB[vm/rss] - 681.59 MB[tc/jemalloc_cache] + 0[reserved] + 0B[waiting_refresh]), limit 56.34 GB, soft limit 50.71 GB. sys available memory 5.96 GB(= 5.96 GB[proc/available] - 0[reserved] - 0B[waiting_refresh]), low water mark 3.13 GB, warning water mark 6.26 GB.
I20241227 13:43:19.897039 62667 daemon.cpp:221] os physical memory 62.60 GB. process memory used 54.59 GB(= 55.46 GB[vm/rss] - 890.54 MB[tc/jemalloc_cache] + 0[reserved] + 0B[waiting_refresh]), limit 56.34 GB, soft limit 50.71 GB. sys available memory 5.76 GB(= 5.76 GB[proc/available] - 0[reserved] - 0B[waiting_refresh]), low water mark 3.13 GB, warning water mark 6.26 GB.

54.06 GB(= 54.83 GB[vm/rss]
内存超过90%之后，开始gc
但是be认为的内存下降的很快，实际使用的内存下降的很慢
有什么办法可以解决一下吗

**Comment by ixzc** [comment]
中文问题建议去中文论坛提问
https://ask.selectdb.com/
